parser/src/parser.cpp

```diff
@@ -147,18 +206,38 @@ int main(int argc, char* argv[])
     return 1;
   }
 
-  //--- Start parsers ---//
+  //--- Check workspace and project directory ---//
+
+  std::string projDir = prepareProjectDir(vm);
```

Comment:
What if the workspace contains multiple projects? Which of these should we insert to the variable map?